startup/trash/999-load.py

```diff
@@ -1,18 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Please restore all of these files.

Further, as we are using git, if you want to delete a file just delete it rather than moving it to a folder called trash.

Reply:
Okay